.gitignore

```diff
@@ -3,8 +3,8 @@
 /vendor/
 /_resources/
 /public/_resources/
+/public/static/[0-9]
```

Comment:
```suggestion
```
As per https://github.com/silverstripe/api.silverstripe.org/pull/135#discussion_r1984060858 this is already covered in `public/static/.gitignore`
Either use the one there, or use the one here - don't use both.

The one there is better because it covers potential future `10` etc - if you choose to remove that one and instead add the rules here, use `/public/static/*` and `!/public/static/.htaccess`.

Reply:
Opps my bad, have removed